Submodels/file_handling.py

```
@@ -78,22 +78,27 @@
+---------
 - V1.0: Initial working version
 - V1.1: Updated test values. Added leading edge coordinate control of centrebody. Added floating point precision of 3 decimals for domain size. Updated input validation logic.
 - V1.1.5: Fixed import logic of the Parameterizations module to handle local versus global file execution. 
+- V1.2.0: Updated class initialization logic and function inputs to enable existing geometry inputs for debugging/validation
+- V1.2.1: Fixed duplicate leading edge coordinate in fileHandling.fileHandlingMTSET.GetProfileCoordinates(). Implemented nondimensionalisation of geometric parameters for both MTSET and MTFLO input files. 
 """
 
 import numpy as np
 import os
-from scipy import interpolate
+from scipy import interpolate, integrate
```

Ruff (0.8.2)

`scipy.integrate` imported but unused

Remove unused import: `scipy.integrate`

(F401)